aws/resource_aws_codebuild_project_test.go

```diff
@@ -615,9 +613,9 @@ func TestAccAWSCodeBuildProject_Source_Auth(t *testing.T) {
 				Config: testAccAWSCodeBuildProjectConfig_Source_Auth(rName, "FAKERESOURCE1", "OAUTH"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAWSCodeBuildProjectExists(resourceName, &project),
-					resource.TestCheckResourceAttr(resourceName, "source.3680505372.auth.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "source.3680505372.auth.2706882902.resource", "FAKERESOURCE1"),
-					resource.TestCheckResourceAttr(resourceName, "source.3680505372.auth.2706882902.type", "OAUTH"),
+					resource.TestCheckResourceAttr(resourceName, "source.0.auth.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "source.0.auth.2706882902.resource", "FAKERESOURCE1"),
```

Comment:
While we are here, it might be good to see if `source` -> `auth` can be `TypeList` and `MaxItems: 1` even if it wasn't picked up by the linter in this case (e.g. is it valid to have more than one `auth` configuration?). At quick glance it seems like [there can only be one](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_CreateProject.html) 💍 

Otherwise we'll need to keep `DisableBinaryDriver: true` for this test for now.

Reply:
updated! as well in `secondary_source` -> `auth`